test/e2e/cypress/support/commands.ts

```diff
@@ -402,6 +402,52 @@ Cypress.Commands.add('recoverApi', ({ email, returnTo }) => {
     })
 })
 
+Cypress.Commands.add('verificationApiExpired', ({ email, returnTo }) => {
+  cy.shortVerificationLifespan()
```

Comment:
This will affect following tests right? Can you maybe also after creating the expired flow, reset the config?

Reply:
Sure, I could, but we do reset it for each test case [here](https://github.com/ory/kratos/blob/master/test/e2e/cypress/integration/profiles/verification/verify/errors.spec.ts#L40)

Reply:
I see, then this is fine :+1: